src/dmangle.d

```diff
@@ -833,8 +833,13 @@ extern (C++) const(char)* mangleExact(FuncDeclaration fd)
 
 extern (C++) void mangleToBuffer(Type t, OutBuffer* buf)
 {
-    scope Mangler v = new Mangler(buf);
-    v.visitWithMask(t, 0);
+    if (t.deco)
+        buf.writestring(t.deco);
```

Comment:
It's also called in objc.d, and I intend to replace the mangleToBuffer() in mtype.d:getTypeInfoIdent() with a call to it. (The `internal` int is always 0.)